opsmngr/agent_api_keys.go

```diff
@@ -0,0 +1,91 @@
+package opsmngr
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	atlas "github.com/mongodb/go-client-mongodb-atlas/mongodbatlas"
+)
+
+const (
+	agentAPIKeysBasePath = "groups/%s/agentapikeys"
+)
+
+type AgentAPIKeysService interface {
+	Create(context.Context, string, AgentAPIKeysRequest) (*AgentAPIKey, *atlas.Response, error)
+	List(context.Context, string) (*[]AgentAPIKey, *atlas.Response, error)
+	Delete(context.Context, string, string) (*atlas.Response, error)
+}
+
+type AgentAPIKeysServiceOp struct {
+	Client atlas.RequestDoer
+}
+
+type AgentAPIKey struct {
+	ID            string  `json:"_id"`
```

Comment:
I do not why but for this call the id is _id. It may be an error in OM

Reply:
it's probably just exposing the internal, that's the default name in mongodb for id fields